includes/user_map.js

```diff
@@ -4,11 +4,39 @@ module.exports = (params) => {
   }).query(ctx => `
 
 -- for each anonymous_id, find a user_id mapped to it (if one exists)
+with anonymous_id_user_id_pairs as (
+select
+  anonymous_id,
+  user_id
+from
+  (
+  select
+    anonymous_id,
+    user_id
+  from
```

Comment:
I don't see why you need this here - identifies contains all the mappings you require, not getting anything new from tracks/pages.

Reply:
Ignore as discussed, as we see plenty of situations where there is a missing identifies call and this corrects for that nicely.